sentry/src/main/java/io/sentry/ISentryClient.java

```diff
@@ -78,6 +79,22 @@ public interface ISentryClient {
     return captureEvent(event, null, hint);
   }
 
+  /**
+   * Captures the feedback.
+   *
+   * @param feedback The feedback to send.
+   * @param hint An optional hint to be applied to the event.
+   * @param scope An optional scope to be applied to the event.
+   * @return The Id (SentryId object) of the event
+   */
+  default @NotNull SentryId captureFeedback(
+      @NotNull Feedback feedback, @Nullable Hint hint, @Nullable IScope scope) {
+    SentryEvent event = new SentryEvent();
+    event.getContexts().setFeedback(feedback);
+
+    return captureEvent(event, scope, hint);
```

Comment:
I've also read in the iOS [docs](https://docs.sentry.io/platforms/apple/guides/ios/user-feedback/#session-replay) that when feedback is capture, we also capture an ongoing replay. So I think you'd have to call `options.getReplayController().captureReplay(false);` in the new `captureFeedback` method.

Reply:
that one is talking about the UI widget, to capture the replay the makes the user open send the feedback.
I don't think we should add it to the API itself, but only when using the widget (which will come later)

Reply:
hm, I'd expect it to work even if I had my own widget I guess... but that's another point to clarify and align on with other sdks maybe?

Reply:
[Added it](https://github.com/getsentry/sentry/issues/88232#issuecomment-2765477810) to the questions 👍

Reply:
added it 👍